eo-maven-plugin/src/test/java/org/eolang/maven/RegisterMojoTest.java

```diff
@@ -91,4 +91,25 @@ void doesNotFailWhenNoStrictNames(@TempDir final Path temp) throws IOException {
             Matchers.is(true)
         );
     }
+
+    @Test
+    void registersInExternalWithZeroVersions(@TempDir final Path temp) throws IOException {
+        new Home(temp).save(
+            new ResourceOf("org/eolang/maven/file-name/abc-def.eo"),
+            Paths.get("src/eo/org/eolang/maven/foo.eo")
+        );
+        final FakeMaven maven = new FakeMaven(temp)
+            .with("sourcesDir", temp.resolve("src/eo").toFile())
+            .execute(new FakeMaven.Register());
+        MatcherAssert.assertThat(
+            maven.external()
```

Comment:
@maxonfjvipon Please, add messages for the both assertions, like explained here:
https://www.yegor256.com/2023/01/19/layout-of-tests.html#assertions